.github/workflows/build.yml

```diff
@@ -76,7 +76,8 @@ jobs:
           p12-file-base64: ${{ secrets.MACOS_CERTIFICATE }}
           p12-password: ${{ secrets.MACOS_CERTIFICATE_PWD }}
       - name: Sign jag on macOS
-        run: /usr/bin/codesign --force -s CLI-code-sign ./jag_macos -v
+        run: |
+          /usr/bin/codesign --force -s "Developer ID Application: Toitware ApS (33DS2ZRDST)" ./jag_macos -v
```

Comment:
Want to go ahead and add the .tgz file right away?

'jag_macos.tgz' containing 'jag' (signed macos binary)

Reply:
It needs to be notarized as well so this is still not good enough. 
I updated the secrets so had to do this change so I didn't break anything.